src/plugins/intel_gpu/src/runtime/ocl/ocl_ext.hpp

```diff
@@ -958,10 +960,19 @@ class UsmMemory {
 
 private:
     void _allocate(void* ptr) {
-        if (!ptr)
-            throw std::runtime_error("[CL ext] Can not allocate nullptr for USM type.");
```

Comment:
I think it would be better to keep some assertion here.

Reply:
'_allocate' is private func. and it's called by 'allocateHost/Shared/Device' only.
Null ptr is checked at '_check_error'.
That exception message can not help for debugging because there is no allocation size, no USM type.